pkg/daemon/ceph/agent/flexvolume/attachment/crd.go

```diff
@@ -46,23 +48,27 @@ func New(context *clusterd.Context) (Attachment, error) {
 
 // Get queries the Volume CRD from Kubernetes
 func (c *crd) Get(namespace, name string) (*rookalpha.Volume, error) {
-	return c.context.RookClientset.RookV1alpha2().Volumes(namespace).Get(name, metav1.GetOptions{})
+	ctx := context.TODO()
```

Comment:
you don't need `ctx` just pass `context.TODO()` directly.

Reply:
Yup done... 
Initial idea was to make changes to the `attachment.Attachment` interface itself (to match it the same with the latest api changes), but later refrained from it.

Reply:
What's done I still see `ctx := context.TODO()`?

Reply:
Corrected... I missed other lines...

Reply:
I still don't see anything